setup.py

```diff
@@ -36,4 +41,40 @@ def run_tests(self):
             self.pytest_args = [arg for arg in self.pytest_args.split() if arg]
             passed_args.extend(self.pytest_args)
 
-        sys.exit(pytest.main(passed_args))
\ No newline at end of file
+        sys.exit(pytest.main(passed_args))
+
+
+def get_install_requires():
+    with open("requirements.txt", "r") as requirements_file:
+        res = requirements_file.readlines()
+        return [req.split(" ", maxsplit=1)[0] for req in res if req]
+
+
+def get_version():
+    with open(os.path.join("thoth", "unresolved_package_handler", "__init__.py")) as f:
+        content = f.readlines()
+
+    for line in content:
+        if line.startswith("__version__ ="):
+            # dirty, remove trailing and leading chars
+            return line.split(" = ")[1][1:-2]
+    raise ValueError("No version identifier found")
+
+
+VERSION = get_version()
+setup(
+    name="thoth-unresolved-package-handler",
+    version=VERSION,
+    description="Service to send/receive message regarding thoth solver.",
```

Comment:
Service to send/receive message regarding failed advisers runs due to unresolved packages.